tests/test_imageset.py

```diff
@@ -5,18 +5,42 @@
 
 import pytest
 import six.moves.cPickle as pickle
+from mock import patch
 
 from scitbx.array_family import flex
 
+import dxtbx.format.FormatHDF5SaclaMPCCD
 import dxtbx.format.image
 import dxtbx.format.Registry
 import dxtbx.tests.imagelist
 from dxtbx.format.FormatCBFMiniPilatus import FormatCBFMiniPilatus as FormatClass
 from dxtbx.imageset import ExternalLookup, ImageSequence, ImageSetData, ImageSetFactory
 from dxtbx.model import Beam, Detector, Panel
+from dxtbx.model.beam import BeamFactory
 from dxtbx.model.experiment_list import ExperimentListFactory
 
 
+def test_single_file_indices(dials_regression):
+    def dummy_beam():
+        return BeamFactory.simple(1.0)
+
+    with patch.object(
```

Comment:
Clearer where `patch.object` comes from, without being excessively more verbose.
```suggestion
    with mock.patch.object(
```

Reply:
Done